backend/data/src/main/kotlin/io/tolgee/dtos/request/prompt/PromptDto.kt

```diff
@@ -0,0 +1,10 @@
+package io.tolgee.dtos.request.prompt
+
+import io.tolgee.model.enums.BasicPromptOption
+
+data class PromptDto(
+  val name: String,
+  val template: String? = null,
+  val providerName: String,
+  var options: List<BasicPromptOption>? = null,
```

Comment:
Some nome suggesting that this is is relevant for the basic mode would be helpful. Don't know it `options` is the right name anyway.